effectful/src/Effectful/Console/ByteString/Lazy.hs

```diff
@@ -0,0 +1,43 @@
+-- | Lifted functions from "Data.ByteString.Lazy" that are related to standard
+-- streams.
+-- Like the original module, you probably want to import this module qualified
+-- to avoid name clashes with the functions provided by "Prelude", e.g.:
+--
+-- > import Data.ByteString.Lazy (ByteString)
+-- > import qualified Data.ByteString.Lazy as LBS
+-- > import Effectful.Console.ByteString.Lazy (Console)
+-- > import qualified Effectful.Console.ByteString.Lazy as Console
+--
+module Effectful.Console.ByteString.Lazy
+  ( module Effectful.Console.Effect
```

Comment:
Can you export `Console` and `runConsole` explicitly (like e.g. `Effectful.FileSystem` does)? IMO it makes for a more readable haddock.

Reply:
Done.